### PR TITLE
remove comparison to bool constants

### PR DESCRIPTION
**Pull request by mcalhoun**
## what
* Remove comparisons to bool constants `true` and `false`

## why
* The code is easier to read without the extra comparison

### CODE REVIEW

File: internal/exec/terraform.go [hunk 1]
```diff
@@ -181,7 +181,7 @@ func ExecuteTerraform(cmd *cobra.Command, args []string) error {
 	// Handle `terraform deploy` custom command
 	if info.SubCommand == "deploy" {
 		info.SubCommand = "apply"
-		if !info.UseTerraformPlan && !u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag) {
+		if !(info.UseTerraformPlan && u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag)) {
```
Comment (aknysh) [inline internal/exec/terraform.go:184]: this changes the logic

https://brilliant.org/wiki/de-morgans-laws/

```
Not (A and B) is the same as Not A or Not B
```

`!info.UseTerraformPlan && !u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag)` will be true id both are false

`!(info.UseTerraformPlan && u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag))` will be true if ANY is false

Comment (mcalhoun) [reply internal/exec/terraform.go]: You are absolutely right...my bad for not catching that!

1ea35bec90028a8b5928c0f3b809bda4766caeb9

**Review by aknysh — APPROVED** [review]

File: internal/exec/terraform.go [hunk 2]
```diff
@@ -181,13 +181,13 @@ func ExecuteTerraform(cmd *cobra.Command, args []string) error {
 	// Handle `terraform deploy` custom command
 	if info.SubCommand == "deploy" {
 		info.SubCommand = "apply"
-		if info.UseTerraformPlan == false && !u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag) {
+		if !info.UseTerraformPlan && !u.SliceContainsString(info.AdditionalArgsAndFlags, autoApproveFlag) {
```
[the remark this answers is not in the document]
Comment (nitrocode) [reply internal/exec/terraform.go:184]: cc: @mcalhoun

**Review by nitrocode — APPROVED** [review]
LGTM

File: internal/exec/terraform.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (aknysh) [reply internal/exec/terraform.go:184]: https://github.com/cloudposse/atmos/pull/159#discussion_r891734187

**Pull request merged**